recipes/usher/build.sh

```diff
@@ -7,7 +7,7 @@ pushd build
 
 cmake -DTBB_DIR=${PWD}/../oneTBB-2019_U9 -DCMAKE_PREFIX_PATH=${PWD}/../oneTBB-2019_U9/cmake -DCMAKE_INSTALL_PREFIX=${PREFIX} ..
 
-make -j
+make -j 1
 
 cp ./usher ${PREFIX}/bin/
```

Comment:
According to https://github.com/yatisht/usher/releases/tag/v0.2.0 there are 2 new utilities (matUtils and matOptimize), should these also be copied to `${PREFIX}/bin/` ?

Reply:
Yes, tested it locally and we should test this. In the latest commit, thanks for the catch!